pkg/server/handler.go

```diff
@@ -41,7 +44,18 @@ func NewHandlers(ctx context.Context, cfg *Config) (http.Handler, error) {
 	httpHandler := middleware.WithLogging(cfg.AppConfig.Logger, mux)
 	httpHandler = middleware.WithProviderToken(cfg.AppConfig.JwtClient, httpHandler, cfg.AppConfig.Logger)
 
+	restCfg, _, err := kube.RestConfig()
+	if err != nil {
+		return nil, fmt.Errorf("building rest config: %w", err)
+	}
```

Comment:
I think this has been created (a couple of times) before this (IIRC it exists in several parts of the Config, and is created by the calling func in cmd/gitops-server/cmd/cmd.go), does it make sense reuse rather than redefine everywhere?

Reply:
good call, changing it. we have it available there we just need to use it

Reply:
done